pkg/Microsoft.ML.OnnxTransform/Microsoft.ML.OnnxTransform.nupkgproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET component for Sonoma</PackageDescription>
+  </PropertyGroup>
```

Comment:
As discussed in the standup meeting, we want to keep `Microsoft.ML.OnnxTransform` from being a stable package. A way to do that is to add this property here:

```xml
<VersionSuffix Condition="'$(StabilizePackageVersion)' == 'true'">$(PreReleaseLabel)</VersionSuffix>
```

Which will ensure that the `Version` will still have `-preview` in it.

Reply:
See https://github.com/dotnet/machinelearning/blob/9258be22e3338dec18075c6e66048b1861533ec8/Directory.Build.props#L61-L66 for how this is determined in the root props file.  When we want to build stable packages for release, we set `$(StabilizePackageVersion)` to `true` for the build.

Reply:
There was another discussion post this mornings call.

The plan is to use this dev nuget for the bug-bash, and assuming bugs are fixed by Thursday and sonoma goes live before tuesday, we'll switch to point to the stable release of the nuget.

build/Dependencies.props:    <MicrosoftMLScoring>1.0.4-dev48825</MicrosoftMLScoring>

---
In reply to: [220290306](https://github.com/dotnet/machinelearning/pull/942#discussion_r220290306) [](ancestors = 220290306)

Reply:
There was another discussion post this mornings call.

The plan is to use this dev nuget for the bug-bash, and assuming bugs are fixed by Thursday and sonoma goes live before tuesday, we'll switch to point to the stable release of the nuget.

build/Dependencies.props:    <MicrosoftMLScoring>1.0.4-dev48825</MicrosoftMLScoring>

---
In reply to: [220290881](https://github.com/dotnet/machinelearning/pull/942#discussion_r220290881) [](ancestors = 220290881)